railties/test/generators/app_generator_test.rb

```diff
@@ -450,6 +450,9 @@ def test_generator_if_skip_system_test_is_given
       assert_no_match(/capybara/, content)
       assert_no_match(/selenium-webdriver/, content)
     end
+    assert_file "config/application.rb" do |contents|
+      assert_match %r{config\.generators\.system_tests = nil}, contents
```

Comment:
Can we add a test to make sure the scaffold generator don't add the system test file?